common/locales/en/pets.json

```diff
@@ -1,10 +1,14 @@
 {
     "pets": "Pets",
+    "activePet": "Active Pet",
+    "noActivePet": "No Active Pet",
```

Comment:
I thought about that, but it is actually very difficult to do. Say you have strings like:

```json
"wizardInfo": "stuff about Mages",
"warriorInfo": "stuff about Warriors",
"healerInfo": "stuff about Healers",
"rogueInfo": "stuff about Rogues",
```

Then you might do something like:

```js
return window.env.t(user.stats.class + 'Info');
```

How do you write the check to pass for that?

You could monkey patch the translation function to keep track of the strings that get passed into it and then run your test suite, and if you have full coverage, all your strings would get used... but that seems like more trouble than it is worth and would require coordination among the different tests suites which would be a big pain. 

You could have the translation function audit itself in production and see which strings never get used, which could also be an interesting way to see which code paths and features aren't being used. But again, that's a pretty large undertaking.

Anyway, you get the idea.